### PR TITLE
simulator: add case about import data

### PR DESCRIPTION
**Pull request by nolouch**
### What problem does this PR solve? 
Add a case about import data. This case used to evaluate the region and leader distribution of a table when importing data.

### What is changed and how it works?
 simulate import data to different table with different speed.

### Check List 

Tests 

 - Unit test
 - Manual test (add detailed scripts or steps below)

The output like:
```
INFO[0060] leader distribute (table2) [store 1]:37.50% [store 2]:5.56% [store 4]:20.83% [store 5]:4.17% [store 6]:4.17% [store 7]:2.78% [store 8]:19.44% [store 9]:5.56% 
INFO[0060] peer distribute (table2) [store 1]:21.46% [store 2]:7.31% [store 4]:20.09% [store 5]:7.76% [store 6]:8.22% [store 7]:6.39% [store 8]:22.37% [store 9]:6.39% 
```

### CODE REVIEW

File: tools/pd-simulator/simulator/raft.go [hunk 1]
```diff
@@ -305,3 +344,63 @@ func generateSplitKey(start, end []byte) []byte {
...
+	if len(left) > 0 {
+		simutil.Logger.Fatalf("Decode key left some bytes: %v", key)
+	}
+	if err != nil {
+		simutil.Logger.Fatalf("Decode key %v meet error: %v", res, err)
+	}
+	return res
+}
+
+func generateMvccSplitKey(start, end []byte) []byte {
```
Comment (disksing) [inline tools/pd-simulator/simulator/raft.go]: emm, add a description about what is it different from generateSplitKey and how it works?

File: tools/pd-simulator/simulator/raft.go [hunk 2]
```diff
@@ -305,3 +341,76 @@ func generateSplitKey(start, end []byte) []byte {
 	key = append(key, ('a'+'z')/2)
 	return key
 }
+
+func mustDecodeMvccKey(key []byte) []byte {
+	left, res, err := table.DecodeBytes(key)
+	if len(left) > 0 {
+		simutil.Logger.Fatalf("Decode key left some bytes: %v", key)
```
Comment (rleungx) [inline tools/pd-simulator/simulator/raft.go]: better to print the key also

File: tools/pd-simulator/simulator/raft.go [hunk 3]
```diff
@@ -305,3 +341,76 @@ func generateSplitKey(start, end []byte) []byte {
...
+		e uint16
+	)
+	res := make([]byte, len(start))
+	for i := 0; i < len(start); i++ {
+		s := start[i]
+		e = uint16(end[i])
+		if borrow {
+			e += 0x100
+		}
+		// if can't divide 2, assume that the lowposition borrows one from the high position, and then can divide 2.
```
Comment (rleungx) [inline tools/pd-simulator/simulator/raft.go]: low position

File: tools/pd-simulator/simulator/cases/import_data.go [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+			table3: 1 * MB,
+			table5: 16 * MB,
+		}
+	}
+	simCase.Events = []EventInner{e}
+
+	// Checker description
+	startTime := time.Now()
+	simCase.Checker = func(regions *core.RegionsInfo) bool {
+		leaderDstb := make(map[uint64]int)
```
Comment (rleungx) [inline tools/pd-simulator/simulator/cases/import_data.go]: How about `leaderDist` and `peerDist`?

File: tools/pd-simulator/simulator/raft.go (hunk 2 above)
Comment (nolouch) [reply tools/pd-simulator/simulator/raft.go]: already print the key.

File: tools/pd-simulator/simulator/raft.go (hunk 1 above)
Comment (nolouch) [reply tools/pd-simulator/simulator/raft.go]: old way not support the encode key and has a bug like `generateSplitKey("a","ab")`, will get “am”

**Comment by nolouch** [comment]
PTAL

**Comment by disksing** [comment]
LGTM.

**Comment by nolouch** [comment]
PTAL

**Pull request merged**